app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class TopicSerializer
+  include JSONAPI::Serializer
+  attributes :id
```

Comment:
@MansurTsutiev why just `id` isn't there `email` and `full_name` attributes to `User` ?

Reply:
good catch
![](https://media.giphy.com/media/Qw4X3FuddXEbshHRHRm/giphy.gif)